cmd/apiserver/main.go

```diff
@@ -202,23 +153,22 @@ func main() {
 
 	// TODO: make the data loader run interval configurable
 	meterOptions := api.RelayMeterOptions{
-		LoadInterval:     time.Duration(options.loadInterval) * time.Second,
-		DailyMetricsTTL:  time.Duration(options.dailyMetricsTTLSeconds) * time.Second,
-		TodaysMetricsTTL: time.Duration(options.todaysMetricsTTLSeconds) * time.Second,
-		MaxPastDays:      time.Duration(options.maxPastDays) * 24 * time.Hour,
+		LoadInterval:     time.Duration(loadInterval) * time.Second,
+		DailyMetricsTTL:  time.Duration(dailyMetricsTTLSeconds) * time.Second,
+		TodaysMetricsTTL: time.Duration(todaysMetricsTTLSeconds) * time.Second,
+		MaxPastDays:      time.Duration(maxPastDays) * 24 * time.Hour,
 	}
-	log.WithFields(logger.Fields{"postgresOptions": postgresOptions, "meterOptions": meterOptions}).Info("Gathered options.")
 
 	backend := backendProvider{
 		PostgresClient:  pgClient,
-		backendApiUrl:   options.backendApiUrl,
-		backendApiToken: options.backendApiToken,
+		backendAPIURL:   backendAPIURL,
+		backendAPIToken: backendAPIToken,
 	}
 	meter := api.NewRelayMeter(&backend, log, meterOptions)
 	http.HandleFunc("/", api.GetHttpServer(meter, log))
 
 	log.Info("Starting the apiserver...")
-	http.ListenAndServe(fmt.Sprintf(":%d", options.port), nil)
+	http.ListenAndServe(fmt.Sprintf(":%d", port), nil)
```

Comment:
Same as previous comments re options struct: We should not eliminate logical grouping of these settings (i.e. `options` struct) to get a group of variables that are not related to each other in code at all, but are actually all part of the configuration of the service.